content/en/docs/concepts/overview/components.md

```diff
@@ -15,6 +15,10 @@ When you deploy Kubernetes, you get a cluster.
 
 This document outlines the various components you need to have
 a complete and working Kubernetes cluster.
+
+Here's the diagram of a Kubernetes cluster with all the components tied together.
+
+![CCM Comp Overview](/images/docs/post-ccm-arch.png)
```

Comment:
Although this diagram is a good indicator of what an overview should look like, it's out of date. For example, “Minions” are now called “Nodes”.

Ideally: draw a new SVG format diagram that is based on the post-CCM architecture. Remove the focus on cloud-controller manager and make sure all the terms are current.

Also, set the image's `alt` attribute to “Components of Kubernetes” or similar.

Reply:
Hello @sftim,

I have added a new image at

[https://github.com/SupriyaSirbi/website/blob/patch-26/static/images/docs/components-of-kubernetes.png](url)

please review the same